streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/state/StateBackendFactory.java

```diff
@@ -0,0 +1,27 @@
+package io.ray.streaming.runtime.state;
+
+import io.ray.streaming.runtime.config.StreamingGlobalConfig;
+import io.ray.streaming.runtime.config.types.StateBackendType;
+import io.ray.streaming.runtime.state.impl.AtomicFsBackend;
+import io.ray.streaming.runtime.state.impl.MemoryContextBackend;
+
+public class StateBackendFactory {
```

Comment:
StateBackend is seems a little heavy? Can the concept is duplicate and confused with state abstraction in streaming processing engine

Reply:
renamed to ContextBackendFactory